docs/Homebrew-homebrew-cask-Maintainer-Guide.md

```diff
@@ -24,7 +24,7 @@ Note that unlike formulae, casks do not consider the `sha256` stanza to be a mea
 
 ## Merging
 
-In general, using GitHub's "Squash and Merge" button is the best way to merge a PR. This can be used when the PR modifies only one cask, regardless of the number of commits or whether the commit message format is correct. When merging using this method, the commit message can be modified if needed. Usually, version bump commit messages follow the form `Update CASK from OLD_VERSION to NEW_VERSION`.
+In general, using GitHub's "Merge" button is the best way to merge a PR. This can be used when the PR modifies only one cask, regardless of the number of commits or whether the commit message format is correct. When merging using this method, the commit message can be modified if needed. Usually, version bump commit messages follow the form `Update CASK from OLD_VERSION to NEW_VERSION`.
```

Comment:
I realise this PR is already merged, but I wanted to clarify whether this paragraph still remains true after changing from 'Squash and Merge' to 'Merge'; as the latter parts of it seem directly related to a squash merge, not a normal merge?

Reply:
Yeah the paragraph isn't quite correct here.